packages/sdk/src/lib/RuntimeArgs.test.ts

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
+import { CLValue } from './CLValue';
+import { RuntimeArgs } from './RuntimeArgs';
+import { decodeBase16 } from './Conversions';
+import { toBytesU32 } from './byterepr';
+
+describe(`RuntimeArgs`, () => {
+  it('should serialize RuntimeArgs correctly', () => {
+    const args = RuntimeArgs.fromMap({
+      foo: CLValue.fromI32(1)
+    });
+    const bytes = args.toBytes();
+    expect(bytes).to.deep.eq(
+      Uint8Array.from([
+        1,
```

Comment:
Could you fix the linter so it doesn't show arrays like that? It should be in one or two lines here.

Reply:
It is the default style of `prettier`, it doesn't provide an option to customize it, so what we can do is to disable it.